solr/datadog_checks/solr/data/conf.yaml.example

```diff
@@ -171,3 +171,22 @@ instances:
     ## This is useful for cluster-level checks.
     #
     # empty_default_hostname: false
+
+## Log Section
+##
+## type - required - Type of log input source (tcp / udp / file / windows_event)
+## port / path / channel_path - required - Set port if type is tcp or udp.
+##                                         Set path if type is file.
+##                                         Set channel_path if type is windows_event.
+## source  - required - Attribute that defines which Integration sent the logs.
+## service - required - The name of the service that generates the log.
+##                      Overrides any `service` defined in the `init_config` section.
+## tags - optional - Add tags to the collected logs.
+##
+## Discover Datadog log collection: https://docs.datadoghq.com/logs/log_collection/
+#
+# logs:
+#   - type: file
+#     path: /var/solr/logs/solr.log
```

Comment:
Are we sure logs are in this location? I would expect something like the below:

```suggestion
#     path: /var/logs/solr.log
```

Reply:
Yes, this is the correct location. It is described in the documentation, and I verified by running an e2e environment for solr: https://lucene.apache.org/solr/guide/8_6/configuring-logging.html#permanent-logging-settings

Reply:
Thanks for the confirmation. definitely a strange move from them but well :)